tests/pact/keysets.repl

```diff
@@ -317,3 +318,17 @@
   "Rotating keyset in wrong namespace fails"
   "Mismatching keyset namespaces"
   (define-keyset "alice.alice-keys" (read-keyset 'bob-keys)))
+
+;; parser tests
+
+(expect
+  "keyset definition parsing is permissive pre-pact-4.4"
+  (read-keyset "TEST <1>.")
+  (read-keyset "TEST <1>."))
```

Comment:
We need to test `define-keyset`, `describe-keyset`, and `keyset-ref-guard` not so much `read-keyset`. `KeySet`s don't care about the name, you just enforce here that `env-data` can have a field formatted like this.